sfdc37/tests/test.bal

```diff
@@ -202,7 +202,7 @@ function testGetQueryResult() {
 
                 while (jsonRes.nextRecordsUrl != null) {
                     log:printDebug("Found new query result set!");
-                    string nextQueryUrl = jsonRes.nextRecordsUrl.toString() ?: "";
+                    string nextQueryUrl = jsonRes.nextRecordsUrl.toString();
```

Comment:
Don't we need to do the null check for jsonRes.nextRecordsUrl then covert to string?
Because if jsonRes.nextRecordsUrl is null, we will get NullReferanceError.